src/core/lifetime.d

```diff
@@ -2119,9 +2178,9 @@ private void moveEmplaceImpl(T)(scope ref T target, return scope ref T source)
 //              "Cannot move object with internal pointer unless `opPostMove` is defined.");
 //    }
 
+    import core.internal.traits;
```

Comment:
Since this import is a local import, can you be selective of what symbols to import?